larq/models_test.py

```diff
@@ -128,6 +128,89 @@ def test_number_as_readable_str_large():
     assert lq.models._number_as_readable_str(1e16) == "1.00E+16"
 
 
+def test_memory_as_readable_str():
+    correct_strings = [  # 2^i bits, from i = 0 to 74
+        "0.12 B",
+        "0.25 B",
+        "0.50 B",
+        "1.00 B",
+        "2.00 B",
+        "4.00 B",
+        "8.00 B",
+        "16.00 B",
+        "32.00 B",
+        "64.00 B",
+        "128.00 B",
+        "256.00 B",
+        "512.00 B",
+        "1.00 KiB",
+        "2.00 KiB",
+        "4.00 KiB",
+        "8.00 KiB",
+        "16.00 KiB",
+        "32.00 KiB",
+        "64.00 KiB",
+        "128.00 KiB",
+        "256.00 KiB",
+        "512.00 KiB",
+        "1.00 MiB",
+        "2.00 MiB",
+        "4.00 MiB",
+        "8.00 MiB",
+        "16.00 MiB",
+        "32.00 MiB",
+        "64.00 MiB",
+        "128.00 MiB",
+        "256.00 MiB",
+        "512.00 MiB",
+        "1.00 GiB",
+        "2.00 GiB",
+        "4.00 GiB",
+        "8.00 GiB",
+        "16.00 GiB",
+        "32.00 GiB",
+        "64.00 GiB",
+        "128.00 GiB",
+        "256.00 GiB",
+        "512.00 GiB",
+        "1.00 TiB",
+        "2.00 TiB",
+        "4.00 TiB",
+        "8.00 TiB",
+        "16.00 TiB",
+        "32.00 TiB",
+        "64.00 TiB",
+        "128.00 TiB",
+        "256.00 TiB",
+        "512.00 TiB",
+        "1.00 PiB",
+        "2.00 PiB",
+        "4.00 PiB",
+        "8.00 PiB",
+        "16.00 PiB",
+        "32.00 PiB",
+        "64.00 PiB",
+        "128.00 PiB",
+        "256.00 PiB",
+        "512.00 PiB",
+        "1.00 EiB",
+        "2.00 EiB",
+        "4.00 EiB",
+        "8.00 EiB",
+        "16.00 EiB",
+        "32.00 EiB",
+        "64.00 EiB",
+        "128.00 EiB",
+        "256.00 EiB",
+        "512.00 EiB",
+        "1,024.00 EiB",
+        "2,048.00 EiB",
+    ]
+
+    for i, correct_string in enumerate(correct_strings):
+        assert lq.models._memory_as_readable_str(2 ** i) == correct_string
+
```

Comment:
I like the idea of having another test that checks if a random amount of bits is covered successfully

Reply:
What do you mean exactly? A hardcoded "12,345 bits = x KiB" test or just checking whether passing any random int does not crash the method?

Reply:
This is also covered by the snapshot tests which have more "random" numbers.